source/configuration_manual/service_configuration.rst

```diff
@@ -216,6 +219,11 @@ Auth master process connects to auth worker processes. It is mainly used by pass
 
    * **chroot** could also be set if possible.
 
+   .. versionchanged:: v2.3.16
+
+   * **service_count=0**, now counts the number of auth requests processed, can be used to cycle the process after so many auth requests. idle workers are always disconnected and stopped.
```

Comment:
I think versionchanged should be indented in the bulletin itself, like:

```
 * **service_count=0** ...

   .. versionchanged:: v2.3.16
```

So it clearly shows that the change applies to service_count, not chroot. This is how it's commonly done elsewhere in lists with versionadded.

Reply:
And maybe a bit clearer: `**service_count** counts the number of processed auth requests. This can be used to cycle the process after the specified number of auth requests (default is unlimited). The worker processes also stop after being idle for idle_kill seconds.`